### PR TITLE
Lint `refined-github.css`

### PR DESCRIPTION
**Pull request by kidonng**
<h1 align="center">Mega Linter II: The Return of #5399</h1>

Changes to-do

### CODE REVIEW

File: source/refined-github.css [hunk 1]
```diff
@@ -35,11 +35,6 @@
 	display: none !important;
 }
 
-/* Hide `New pull request button` near file list */
-.file-navigation .new-pull-request-btn {
-	display: none;
-}
-
```
Comment (kidonng) [inline source/refined-github.css:42]: Cannot attach screenshots to commits so here's a review comment instead.

The "New pull request button" used to be like:



Now:

Screen Shot 2022-02-27 at 13 32 56

No need to hide it to make space for other features anymore

File: source/refined-github.css [hunk 2]
```diff
@@ -35,17 +35,6 @@
...
-@keyframes fade-in {
-	from {
-		opacity: 0%;
-	}
-}
```
Comment (kidonng) [inline source/refined-github.css:47]: Added in #735 but no longer used (I didn't bother to find when it was removed)

And we are actually overriding the native one:

```css
/* https://github.githubassets.com/assets/node_modules/@primer/css/utilities/animations.scss */
@keyframes fade-in {
  0% {
    opacity: 0;
  }

  100% {
    opacity: 1;
  }
}
```

**Comment by fregante** [comment]
If you want to submit small changes, feel free to push them to a branch and then open a PR when ready for discussion or close to merge.

**Comment by fregante** [comment]
One more lint would be to move the tests out of test and next to the related helpers, where possible, e.g.

- test/github-url.ts -> source/github-helpers/github-url.**test.**ts

**Pull request closed without merging**